cache

```diff
@@ -201,6 +199,8 @@ cache::lftp() {
   command_log=$(echo -e "$lftp_command" | lftp -u $SEMAPHORE_CACHE_USERNAME, sftp://$SEMAPHORE_CACHE_URL 2>&1)
   command_status=$?
 
+  echo $command_log
```

Comment:
We should use `cache::log` to have consistent, timestamped outputs throughout the script.